escrow/contracts/Escrow.cdc

```diff
@@ -231,6 +232,22 @@ access(all) contract Escrow {
         }
     }
 
+    // Handler for depositing NFTs to the Escrow Collection, used by the NFTLocker contract.
+    access(all) struct DepositHandler: NFTLocker.IAuthorizedDepositHandler {
```

Comment:
Deposit handler definition to unlock a locked NFT and add into escrow leaderboard